app/controllers/flight_api_service_controller.rb

```diff
@@ -0,0 +1,12 @@
+require './config/environment'
+require './app/service/connection'
+
+class FlightApiServiceController < Sinatra::Base
+
+  get '/flights' do
+    response = FlightService.flights_search(params)
+    status response.status
+    body response.body
```

Comment:
Status and body are used to specifically return response.status and response.body

Reply:
Nice! We'll probably be doing some data manipulation, but good start